src/modules/gimbal/gimbal.cpp

```diff
@@ -568,5 +618,6 @@ Test the output by setting a angles (all omitted axes are set to 0):
 	PRINT_MODULE_USAGE_ARG("<sysid> <compid>", "MAVLink system ID and MAVLink component ID", false);
 	PRINT_MODULE_USAGE_COMMAND_DESCR("test", "Test the output: set a fixed angle for one or multiple axes (gimbal must be running)");
 	PRINT_MODULE_USAGE_ARG("roll|pitch|yaw <angle>", "Specify an axis and an angle in degrees", false);
+	PRINT_MODULE_USAGE_ARG("rollrate|pitchrate|yawrate <angle rate>", "Specify an axis and an angle rate in degrees / second", false);
```

Comment:
module docs ci fails to extract the `yawrate` here.
We should make sure this `split` also splits by space: https://github.com/PX4/PX4-Autopilot/blob/main/Tools/px4moduledoc/srcparser.py#L445

Reply:
Thanks, I don't quite understand why but it seems to fix the issue indeed :).

Reply:
It's a simple consistency check that parses the module docs and tries to ensure all parsed arguments in the code are actually documented.